ctl/src/command.rs

```diff
@@ -1319,8 +1325,8 @@ pub fn query_application(mut channel: Channel<CommandRequest,CommandResponse>, j
           if let Some(needle) = application_id.or(domain) {
             if let Some(CommandResponseData::Query(data)) = message.data {
               if json {
-                print_json_response(&data);
-                return;
+                print_json_response(&data)?;
+                return Ok(());
```

Comment:
return print_json_response()? Maybe it makes this less readable

Reply:
good idea yeah